exercises/acronym/.meta/src/reference/java/Acronym.java

```diff
@@ -13,14 +13,12 @@ String get() {
         return acronym;
     }
 
-    private String generateAcronym(String phrase){
-        final Pattern BREAK_WORDS = Pattern.compile("[A-Z]+[a-z]*|[a-z]+");
-        final Matcher matcher = BREAK_WORDS.matcher(phrase);
-        final StringBuilder stringBuilder = new StringBuilder();
-        while (matcher.find()){
-            stringBuilder.append(matcher.group().charAt(0));
-        }
-        return stringBuilder.toString().toUpperCase();
+    private String generateAcronym(String phrase) {
+        return Arrays.stream(phrase.split("\\W"))
```

Comment:
a non-word character consists of `a-z`, `A-Z`, `0-9` & `_`. This may potentially be buggy, since the existing code only considers `a-z` & `A-Z`. Maybe I should use `\p{Alpha}` instead.